build/prepare-sa-test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+kubectl delete namespace namespace-for-testing
```

Comment:
Have you try to make namespace configurable?

Reply:
Do you have any specific application in mind?
It could be an improvement anyway, let me see if there are impacts in the code.

Reply:
done